src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodsService.scala

```diff
@@ -1,60 +1,35 @@
 package org.broadinstitute.dsde.firecloud.service
 
-import akka.actor.{Actor, _}
-import com.wordnik.swagger.annotations.{Api, _}
-import org.broadinstitute.dsde.firecloud.MethodsClient
-import org.broadinstitute.dsde.firecloud.model.MethodEntity
+import akka.actor.{Actor, Props}
+import com.wordnik.swagger.annotations.{Api, ApiOperation}
 import org.slf4j.LoggerFactory
+import spray.client.pipelining._
 import spray.routing.{HttpService, Route}
 
+import org.broadinstitute.dsde.firecloud.{FireCloudConfig, HttpClient}
+
 class MethodsServiceActor extends Actor with MethodsService {
   def actorRefFactory = context
   def receive = runRoute(routes)
 }
 
-@Api(value = "/methods", description = "Methods Service", produces = "application/json")
+@Api(value = "/methods", description = "Methods Service", produces = "application/json, text/plain")
```

Comment:
I don't think it is true that text/plain can be returned from agora for the ```/methods``` call. It can be in the get ```/methods/{namespace}/{name}/{snapshotId}``` call, but I think it would be better to annotate it on that service call.